addons/vscode/src/extension.ts

```diff
@@ -15,17 +15,17 @@ async function loadHTMLFile(context: vscode.ExtensionContext, relativePath: stri
 	return fileContents;
 }
 
-export async function getTypstWsPath(extensionPath?: string): Promise<string> {
-	const state = getTypstWsPath as unknown as any;
-	(!state.BINARY_NAME) && (state.BINARY_NAME = "typst-ws");
+export async function getTypstPreviewPath(extensionPath?: string): Promise<string> {
```

Comment:
When we names it by `TypstWs`, we can know it definitely refer to that cli program. But `TypstPreview` is ambiguous since it is also the name of extension.

I would suggest to remove `TpstPreview` prefixes and rename it as `getCliProgramPath`.